### PR TITLE
[Outlook] (snippet) Add note to sample about mode restriction

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Addresses documentation bug in issue https://github.com/OfficeDev/office-js-docs-reference/issues/332.

Also gets latest d.ts and snippets changes.

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[40a9e0c](https://github.com/OfficeDev/office-js-docs-reference/pull/340/commits/40a9e0cf6c2e9a82b0362d0dd3734cb7dc0dc7f5)_: 

:clock10: **Preparing**: average preparing time is _1 min(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[40a9e0c](https://github.com/OfficeDev/office-js-docs-reference/pull/340/commits/40a9e0cf6c2e9a82b0362d0dd3734cb7dc0dc7f5)_: 

:clock10: **Incremental building**: average incremental building time is _4 min(s) 48 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[40a9e0c](https://github.com/OfficeDev/office-js-docs-reference/pull/340/commits/40a9e0cf6c2e9a82b0362d0dd3734cb7dc0dc7f5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.removeduplicatesresult.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.removeduplicatesresult.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.shape.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.shape.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.shapecollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.shapecollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.textframe.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.textframe.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_1/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_2/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_3/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_4/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_5/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_6/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_7/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-excel-release/excel.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/api-extractor-inputs-excel-release/excel.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/script-lab-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/script-lab-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201903012322490979-340%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zfutDSaHK6%2B94SWftQawsEoXP0wa0hvf9xPBRng7TGk%3D&st=2019-03-01T23%3A23%3A43Z&se=2019-04-01T23%3A28%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/code-snippets/outlook-snippets.yaml [hunk 1]
```diff
@@ -819,6 +819,8 @@ Office.CustomProperties.saveAsync:
     }
 Office.EmailAddressDetails.appointmentResponse:
   - |-
+    // The following sample provides the responses of required attendees.
+    // Note that Appointment Read/Attendee mode is required.
```
Comment (AlexJerabek) [inline docs/code-snippets/outlook-snippets.yaml]: Consider "Read/Attendee" -> "Read (Attendee)"

File: docs/code-snippets/outlook-snippets.yaml [hunk 2]
```diff
@@ -819,6 +819,8 @@ Office.CustomProperties.saveAsync:
     }
 Office.EmailAddressDetails.appointmentResponse:
   - |-
+    // The following sample provides the responses of required attendees.
```
Comment (AlexJerabek) [inline docs/code-snippets/outlook-snippets.yaml]: "responses of required attendees" -> "responses from required attendees"

File: docs/code-snippets/outlook-snippets.yaml (hunk 1 above)
Comment (AlexJerabek) [inline docs/code-snippets/outlook-snippets.yaml]: Is it that the mode is required or that you're required to be *in* the mode?

**Comment by o365devx** [comment]
OPS Build status updates of commit _[8ce16f5](https://github.com/OfficeDev/office-js-docs-reference/pull/340/commits/8ce16f55fc5e6b40c219a75ed3fb41d66f5fb747)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[8ce16f5](https://github.com/OfficeDev/office-js-docs-reference/pull/340/commits/8ce16f55fc5e6b40c219a75ed3fb41d66f5fb747)_: 

:clock10: **Incremental building**: average incremental building time is _4 min(s) 48 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[8ce16f5](https://github.com/OfficeDev/office-js-docs-reference/pull/340/commits/8ce16f55fc5e6b40c219a75ed3fb41d66f5fb747)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.removeduplicatesresult.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.removeduplicatesresult.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.shape.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.shape.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.shapecollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.shapecollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.textframe.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/excel/excel.textframe.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.addincommands.event.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/office/office.addincommands.event.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.item.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.item.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_1/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_2/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_3/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_4/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_5/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_6/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/docs/docs-ref-autogen/outlook_1_7/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-excel-release/excel.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/api-extractor-inputs-excel-release/excel.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/api-extractor-inputs-office/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook/outlook.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/api-extractor-inputs-outlook/outlook.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/office.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/json/office.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/outlook.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/json/outlook.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office_preview.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/office_preview.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/script-lab-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-sample-restriction/generate-docs/script-inputs/script-lab-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201903042118198974-340%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=z15d4JQBQbPnUG0Q1Yk8eQY2bQGeBCphPsAjr85dMoo%3D&st=2019-03-04T21%3A19%3A00Z&se=2019-04-04T21%3A24%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**